packages/ui/src/Text.tsx

```diff
@@ -259,5 +200,15 @@ export const Text = styled(TextComponent)<TextProps>`
         return 'center'
     }
   }};
+  ${({ align }) => {
```

Comment:
this is wierd, there is no prop before this like justify-content, or text-align, so this will not work as css.

Reply:
text-align works with _align && `text-align: ${align}_ for left, right and center. Should I leave it and delete justify-content?

Reply:
yes but This will just render `flex-start` I get the previous lines like `align & text-align: ${align}`. But this makes no sense. If you see what I see :)